mocks/wasmd/x/wasm/keeper/ante.go

```diff
@@ -0,0 +1,72 @@
+package keeper
+
+// https://github.com/CosmWasm/wasmd/blob/v0.22.0/x/wasm/keeper/ante.go
```

Comment:
Copied the code from here. This was needed for integration tests to work with experimental.